device/core/src/pnp/properties.ts

```diff
@@ -0,0 +1,81 @@
+import { JSONValue, JSONObject } from '.';
+import { Twin } from '../twin';
+
+export function generateWritablePropertyResponse(value: JSONValue, ackCode: number, ackDescription: string, ackVersion: number): {
+    value: JSONValue,
+    ac: number,
+    ad: string,
+    av: number
+} {
+    return {
+        value,
```

Comment:
>value [](http://example.com/codeflow?start=8&length=5)

the schema specifies that you tag it with the name `value`.  e.g. `return { value: value, `.  Is this name somehow implied in JS or TS syntax when you use it like this?

Reply:
This is a feature called Object Literal Shorthand Syntax:
https://alligator.io/js/object-property-shorthand-es6/

Reply:
Cool!  I assume that our typescript config correctly transpiles ES6/ES2015 (or doesn't need to transpile this because of our supported Node versions).

Reply:
Yep, it transpiles down to ES5